### DISCUSSION TITLE
Twc doesnt work with Radix tabs

### DISCUSSION THREAD

**Discussion by elmarvr** [gregberge/twc#36] [Q&A]
The providers break when using twc with the radix tabs component

**Comment by gregberge** [comment]
@elmarvr could you elaborate please?

**Comment by elmarvr** [comment]
Yeah the tabs break when i use twc(Tabs.Root)`bg-red-500`

**Comment by gregberge** [comment]
I don't see any problem. https://codesandbox.io/p/devbox/recursing-sea-mnc7x5?file=%2FApp.jsx%3A4%2C33

**Comment by elmarvr** [comment]
That's weird I can't get it to work. I'll double check and create a code snippet if it still doesn't work
